dbld/rules

```diff
@@ -21,6 +21,8 @@ TARBALL=$(BUILD_DIR)/syslog-ng-$(VERSION).tar.gz
 MODE=snapshot
 CONFIGURE_OPTS=--enable-debug --enable-manpages --with-python=2 --prefix=/install --with-docbook=/usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl $(CONFIGURE_ADD)
 
+-include dbld/rules.conf
```

Comment:
Sorry if I missed it, but I don't see this rules.conf file

Reply:
That file can be added locally so you can customize your dbld related options (for me, I override the default docker image to be used and the default bootstrap arguments).

very handy.